### ISSUE TITLE
Apply Aspect globally in AssemblyInfo.cs

### ISSUE THREAD

**Issue by JamesFlames** [vescon/MethodBoundaryAspect.Fody#50]
Is it still possible to apply Aspects globally at assembly level?
How is this done? Can wildcards be used to apply Aspects just to a namespace within an assembly?

**Comment by marcells** [comment]
Yes, it is possible, but cannot be restricted to namespaces. It generally applies to all public methods and properties.

@Ralf1108 Could you confirm this information and maybe update the documentation?

**Comment by Ralf1108** [comment]
Just add the following line to set your desired Aspect to be weaved for the whole assembly.
You can omit the "Attribute"-Suffix.

`[assembly: Log]`

**Comment by Ralf1108** [comment]
To support namespace filters we take a PR ;-)
